### PR TITLE
move server items list to content

### PR DESCRIPTION
**Pull request by dantavori**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8900

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -278,7 +278,7 @@ jobs:
         - run:
             name: Checkout the Content Repo
             command: |
-              git clone https://github.com/demisto/content.git
+              git clone https://github.com/demisto/content.git --branch server-items-list
```
Comment (dantavori) [inline .circleci/config.yml:281]: added for testing, will remove before merging

File: demisto_sdk/commands/content_graph/common.py [hunk 2]
```diff
@@ -321,510 +328,17 @@ def _lazy_decorator(self):
...
-        "mysql",
-        "mssql",
-        "google",
-        "crowdstrike-streaming-api",
-        "kafka",
-        "syslog",
-        "fcm",
-    ],
-}
+def get_server_content_items() -> Dict[ContentType, list]:
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/content_graph/common.py:331]: doc string

File: .circleci/config.yml (hunk 1 above)
Comment (ShahafBenYakir) [reply .circleci/config.yml:281]: remove before merging

**Review by ShahafBenYakir — APPROVED** [review]
Shtak

**Pull request merged**